cli/task_runner.rs

```diff
@@ -263,16 +275,16 @@ impl ShellCommand for NodeCommand {
   ) -> LocalBoxFuture<'static, ExecuteResult> {
     // run with deno if it's a simple invocation, fall back to node
     // if there are extra flags
-    let mut args = Vec::with_capacity(context.args.len());
+    let mut args: Vec<OsString> = Vec::with_capacity(context.args.len());
     if context.args.len() > 1
```

Comment:
Side note: This seems wrong. Shouldn't this be `> 0` because it's only checking the first argument?

Reply:
I'll do it in a separate PR.